### DISCUSSION TITLE
쉐이더 Standard_js 의 tonemapping 부분 주석 제거해도 괜찮나요?

### DISCUSSION THREAD

**Discussion by earthboom** [naverz/zepeto-studio-kor#468] [5. Q&A]
안녕하세요.
제공해주신 Standard_js 쉐이더 코드 중에

void tonemapping (Input IN, SurfaceOutputStandardSpecular o, inout fixed4 color) {
            // color = ApplyColorGrading(color);
        }

이 부분의 주석을 제공하여 해당 기능을 적용해도 되나요?
주석을 제거하여 원하는 효과는 얻었지만, 기본적으로 주석이 되어 있어, 혹시나 해서 문의 드립니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요~주석을 제거해서 사용하셔도 됩니다^^
공식적으로 올려드린 샘플 파일은 자유롭게 응용하면서 사용해 주세요! 😄
